src/vorta/views/archive_tab.py

```diff
@@ -44,6 +51,8 @@
 
 logger = logging.getLogger(__name__)
 
+PRECISION = 1
```

Comment:
Please add a docstring for the constant.

Reply:
Not sure how to do that, only way I found is the Annotated typing, but that needs python 3.9 at least (https://stackoverflow.com/a/8820636/527935). So instead I renamed the constant to `SIZE_DECIMAL_DIGITS` which seems more clear

Reply:
Sphinx allows documenting python variables. I always use [their `#:`-format](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#directive-autoattribute).

Reply:
ok. I still find the new name self explanatory enough, don't you think?

Reply:
Method, function and class names should be self-explanatory too. However docstrings are mandatory still.

Reply:
ok done. If they are mandatory then please update the Contributing guide.

Reply:
> If they are mandatory then please update the Contributing guide.

Quoting the docs:

> Write docstrings at least for methods and functions. Docstrings for classes, attributes and global variables are very welcome too.

So docstrings are mandatory for methods and functions, although you are right that I didn't write the same about other docstrings in the guide. In these cases I will request them as applicable.